src/backend/distributed/utils/statistics_collection.c

```diff
@@ -86,8 +100,138 @@ CollectBasicUsageStatistics(void)
 	escape_json(fields, unameData.machine);
 	appendStringInfoString(fields, "}");
 
-	return SendHttpPostJsonRequest(STATS_COLLECTION_HOST "/v1/usage_reports",
-								   fields->data, HTTP_TIMEOUT_SECONDS);
+	headers = curl_slist_append(headers, "Content-Type: application/json");
+	headers = curl_slist_append(headers, "charsets: utf-8");
+	success = SendHttpRequest(STATS_COLLECTION_HOST "/v1/usage_reports",
+							  headers, fields->data, HTTP_TIMEOUT_SECONDS,
+							  NULL /* no callbacks */);
+	curl_slist_free_all(headers);
+
+	return success;
+}
+
+
+/* CheckForUpdates queries Citus servers for newer releases of Citus. */
+void
+CheckForUpdates(void)
+{
+	SendHttpRequest(
```

Comment:
Ignoring the return value. I don't know how we can use for it as timing of check for updates is same as statistics collection and I thought success/failure of statistics collection should rule there.